src/core/expr/fexpr.cc

```diff
@@ -510,103 +588,16 @@ DECLARE_METHOD(&PyFExpr::shift)
     ->arg_names({"n"})
     ->n_positional_or_keyword_args(1);
 
-
-
-oobj PyFExpr::last(const XArgs&) {
-  auto lastFn = oobj::import("datatable", "last");
-  return lastFn.call({this});
-}
-
-DECLARE_METHOD(&PyFExpr::last)
-    ->name("last")
-    ->docs(dt::doc_FExpr_last);
-
-
-
-oobj PyFExpr::count(const XArgs&) {
-  auto countFn = oobj::import("datatable", "count");
-  return countFn.call({this});
-}
-
-DECLARE_METHOD(&PyFExpr::count)
-    ->name("count")
-    ->docs(dt::doc_FExpr_count);
-
-
-
-oobj PyFExpr::first(const XArgs&) {
-  auto firstFn = oobj::import("datatable", "first");
-  return firstFn.call({this});
-}
-
-DECLARE_METHOD(&PyFExpr::first)
-    ->name("first")
-    ->docs(dt::doc_FExpr_first);
-
-
-oobj PyFExpr::as_type(const XArgs& args) {
-  auto as_typeFn = oobj::import("datatable", "as_type");
-  oobj new_type = args[0].to_oobj();
-  return as_typeFn.call({this, new_type});
-}
-
-
-DECLARE_METHOD(&PyFExpr::as_type)
-    ->name("as_type")
-    ->docs(dt::doc_FExpr_as_type)
-    ->arg_names({"new_type"})
-    ->n_positional_args(1)
-    ->n_required_args(1);
-
-
-oobj PyFExpr::nunique(const XArgs&) {
-  auto nuniqueFn = oobj::import("datatable", "nunique");
-  return nuniqueFn.call({this});
-}
-
-DECLARE_METHOD(&PyFExpr::nunique)
-    ->name("nunique")
-    ->docs(dt::doc_FExpr_nunique);
-    
-
-oobj PyFExpr::countna(const XArgs&) {
-  auto countnaFn = oobj::import("datatable", "countna");
-  return countnaFn.call({this});
-}
-
-DECLARE_METHOD(&PyFExpr::countna)
-    ->name("countna")
-    ->docs(dt::doc_FExpr_countna);
-
-
-oobj PyFExpr::cumsum(const XArgs&) {
-  auto cumsumFn = oobj::import("datatable", "cumsum");
-  return cumsumFn.call({this});
-}
-
-DECLARE_METHOD(&PyFExpr::cumsum)
-    ->name("cumsum")
-    ->docs(dt::doc_FExpr_cumsum);
-
-
-oobj PyFExpr::cummin(const XArgs&) {
-  auto cumminFn = oobj::import("datatable", "cummin");
-  return cumminFn.call({this});
+oobj PyFExpr::sum(const XArgs&) {
+  auto sumFn = oobj::import("datatable", "sum");
+  return sumFn.call({this});
 }
 
-DECLARE_METHOD(&PyFExpr::cummin)
-    ->name("cummin")
-    ->docs(dt::doc_FExpr_cummin);
+DECLARE_METHOD(&PyFExpr::sum)
+    ->name("sum")
+    ->docs(dt::doc_FExpr_sum);
```

Comment:
Oh, that was just a request to arrange the lists. No need to worry about all the function implementations, otherwise we will need to move so much code :)